packages/platform/types/organizations/organization_users_2024_06_18/inputs/get-organization-users.input.ts

```diff
@@ -0,0 +1,3 @@
+import { GetUsersInput_2024_06_18 } from "@calcom/platform-types";
+
+export class GetOrganizationUsersInput_2024_06_18 extends GetUsersInput_2024_06_18 {}
```

Comment:
I don't think we should add version number for the base endpoints, we only add the api version date within files/folders when we absolutely need to do a breaking change

Reply:
@supalarry what are your thoughts on this? I know you mentioned to follow the same conventions as https://github.com/calcom/cal.com/pull/15457

Reply:
I am not relligious about software - not defining the version is good too in this case.